test/unittests/execute_test.cpp

```diff
@@ -85,6 +85,40 @@ TEST(execute, call_with_arguments)
     EXPECT_EQ(ret[0], 0x2);
 }
 
+TEST(execute, call_indirect)
+{
+    /*
+      (type $out-i32 (func (result i32)))
```

Comment:
I'd refrain from using dashes in names.

Reply:
That's taken from spec tests (a lot of dashes there), but I can change it.

Reply:
Ah ok, I had a feeling those would be invalid (as they are in most languages). But yeah, since it is s-expressions/lisp inspired, dashes are accepted there.